ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -500,8 +500,14 @@ static int HandleFileOpOperation(
             goto CleanupAndReturn;
         }
 
+        // We used to assert if we couldn't read the attributes of a file here.
```

Comment:
I don't think it's very useful to write a comment that says what the code "used to" do. These sorts of comments just end up being confusing later on.